examples/consumer-and-issuer/src/actions/index.ts

```diff
@@ -1,99 +1,101 @@
 "use server";
-import type { idOSCredential } from "@idos-network/credentials";
+import type {
+  AvailableIssuerType,
+  CredentialFields,
+  CredentialSubject,
+  idOSCredential,
+} from "@idos-network/credentials";
 import { base64Decode, base64Encode, hexDecode, toBytes } from "@idos-network/utils/codecs";
 import type { EncryptionPasswordStore } from "@idos-network/utils/enclave";
+import countries3to2 from "countries-list/minimal/countries.3to2.min.json";
 import jwt from "jsonwebtoken";
 import invariant from "tiny-invariant";
 import nacl from "tweetnacl";
 import { idOSConsumer } from "@/consumer.config";
 import { idOSIssuer } from "@/issuer.config";
 
-// biome-ignore lint/suspicious/noExplicitAny: We will use `any` to avoid type errors
-const vcTemplate = (kycData: Record<string, any>) => {
-  const { info, review, agreement } = kycData;
-  const document = info.idDocs[0];
+const generateCredentials = async (
+  issuer: Awaited<ReturnType<typeof idOSIssuer>>,
+  // biome-ignore lint/suspicious/noExplicitAny: We will use `any` to avoid type errors
+  kycData: Record<string, any>,
+  idFrontFile: Buffer,
+  selfieFile: Buffer,
+) => {
+  const { info, review, email, phone } = kycData;
+  const idDoc = info.idDocs[0];
 
-  const body = {
-    xsd: "http://www.w3.org/2001/XMLSchema#",
-    aux: "https://raw.githubusercontent.com/trustfractal/claim-schemas/master/aux.xml",
+  // TODO: Fix this
+  const issuerHost = "https://consumer-and-issuer-demo.playground.idos.network";
```

Comment:
@ditoglez what should we use here?

Reply:
because `process.env.VERCEL_URL` can't be used... it's bind to one ... but we are in demo... so maybe it's not so bad